src/lib/viewers/media/MediaBaseViewer.js

```diff
@@ -584,10 +579,20 @@ class MediaBaseViewer extends BaseViewer {
             this.setMediaTime(start);
         }
         if (arguments.length === 0 || hasValidStart) {
-            this.mediaEl.play();
+            // Play may return a promise depending on browser support. This promise
+            // will resolve when playback starts.
+            // https://webkit.org/blog/7734/auto-play-policy-changes-for-macos/
+            const playPromise = this.mediaEl.play();
             this.handleRate();
             this.handleVolume();
+
+            if (playPromise && typeof playPromise.then === 'function') {
```

Comment:
Could condense this to `return (playPromise && typeof playPromise.then === 'function') ? playPromise : Promise.reject(new Error(PLAY_PROMISE_NOT_SUPPORTED))`?